src/qt/frame.cpp

```diff
@@ -208,22 +208,23 @@ QScrollArea *wxFrame::QtGetScrollBarsContainer() const
 
 void wxFrame::DoGetClientSize(int *width, int *height) const
 {
-    wxWindow::DoGetClientSize(width, height);
-
-    // Adjust the height, taking the status and menu bars into account, if any:
-    if ( height )
+    QMainWindow *main_window = GetQMainWindow();
```

Comment:
`main_window` does not respect the naming convention, but more important, is it always a valid pointer? i.e. even for a `wxFrame` that was not `Create()`ed?

Reply:
This will definitely crash if called before `Create()`.

There should be a `wxCHECK_MSG( mainWindow, "window should be created" )` here for consistency with e.g. wxGTK.